_posts/2017/2017-10-06-epages-at-euruko.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "ePages at EuRuKo"
+date: "2017-09-27 09:34:02"
+image: blog-header/budapest-525857_1920.jpg
+image-position: top
+categories: events
+authors: ["Unai"]
+---
+
+Last week our team (Internal tools team) attended the European Ruby Conference in Budapest. Although we use more languages other than Ruby, we love it. It's the base of our team, so it felt natural be there.
```

Comment:
Last week my team (Internal tools team)..... Although we use other languages, we love working with Ruby. It's the base of our team, so it felt natural to be there.